src/App.js

```diff
@@ -1,29 +1,16 @@
-import logo from './logo.svg';
+import React from 'react';
+import Calculator from './components/Calculator';
 import './App.css';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          {' '}
-          <code>src/App.js</code>
-          {' '}
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+//  eslint-disable-next-line react/prefer-stateless-function
```

Comment:
- [ ] since you are disabling this line of ode you should be aware that a stateless function is that you did not add a constructor and props.